### ISSUE TITLE
Unable to go get k8s.io/client-go/kubernetes/typed/core/v1beta1

### ISSUE THREAD

**Issue by jkondapalli03** [kubernetes/client-go#552]
I'm trying to use statefulsets interfaces. I tried go get & dep ensure to get v1beta1 pkg into my environment. Nothing works...
- go get k8s.io/client-go/kubernetes/typed/core/v1beta1 fails with the following error
`└─ $ ▶ go get k8s.io/client-go/kubernetes/typed/core/v1beta1
package k8s.io/client-go/kubernetes/typed/core/v1beta1: cannot find package "k8s.io/client-go/kubernetes/typed/core/v1beta1" in any of:
        /usr/local/go/src/k8s.io/client-go/kubernetes/typed/core/v1beta1 (from $GOROOT)
        /home/*****/.go/src/k8s.io/client-go/kubernetes/typed/core/v1beta1 (from `$GOPATH)`

- `go get k8s.io/client-g`o or `go get k8s.io/client-go/... ` silently fails
- `dep ensure -v` fails with following error:
`ensure Solve(): No versions of k8s.io/client-go met constraints:
        v10.0.0: Could not introduce k8s.io/client-go@v10.0.0, as its subpackage k8s.io/client-go/kubernetes/typed/core/v1beta1 is missing. (Package is required by (root).)
        v9.0.0: Could not introduce k8s.io/client-go@v9.0.0, as it is not allowed by constraint ^10.0.0 from project github.com/my-org/my-project.`

**Comment by mnkg561** [comment]
I also started seeing this error since this after noon.

```Solving failure: No versions of k8s.io/client-go met constraints:
        v10.0.0: Could not introduce k8s.io/client-go@v10.0.0, as it requires package k8s.io/api/admissionregistration/v1alpha1 from k8s.io/api, but in version master that package is missing.
        v9.0.0: Could not introduce k8s.io/client-go@v9.0.0, as it is not allowed by constraint ^10.0.0 from project github.intuit.com/cloud-security/k8s-pod-scalar.
        v8.0.0: Could not introduce k8s.io/client-go@v8.0.0, as it is not allowed by constraint ^10.0.0 from project github.intuit.com/cloud-security/k8s-pod-scalar.```

Any changes happened today (which can effect projects which uses "dep" to start seeing this error)?

**Comment by liggitt** [comment]
> * No versions of k8s.io/client-go met constraints: v10.0.0: Could not introduce k8s.io/client-go@v10.0.0, as its subpackage k8s.io/client-go/kubernetes/typed/core/v1beta1 is missing.

What is referencing package `k8s.io/client-go/kubernetes/typed/core/v1beta1`? That package does not exist in client-go (and never has): https://github.com/kubernetes/client-go/tree/release-10.0/kubernetes/typed/core


> Any changes happened today (which can effect projects which uses "dep" to start seeing this error)?

Yes, that is the result of using a master version of `k8s.io/api` with a release branch version of client-go. See https://github.com/kubernetes/client-go/issues/551#issuecomment-457977525


/close

**Comment by jkondapalli03** [comment]
Yup! the package does not exist and it is my fault to open an issue with the wrong package information. I was actually looking for apps/v1beta1. 

I resolved this issue my self by using appropriate versions & packages.
